docs/get-started/index.md

```diff
@@ -0,0 +1,313 @@
+---
+title: Get started with Storybook
+hideRendererSelector: true
+---
+
+<!-- prettier-ignore-start -->
+
+export const RightArrow = () => (
+  <svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M4.896 10.146a.5.5 0 00.708.708l3.5-3.5a.5.5 0 000-.708l-3.5-3.5a.5.5 0 10-.708.708L8.043 7l-3.147 3.146z" fill="currentColor"></path></svg>
+);
+
+<!-- prettier-ignore-end -->
+
+Welcome to Storybook's documentation ✦ Learn how to get started with Storybook in your project. Then explore the main concepts and additional resources to help you grow and maintain your Storybook.
```

Comment:
Small tweaks:

- clarify ‘main concepts’ to be ‘Storybook’s main concepts’
- break up the final sentence so that there’s a verb more clearly attached to the second part (I think it makes it easier to follow)

Suggested change:
`Welcome to Storybook's documentation ✦ Learn how to get started with Storybook in your project. Then, explore Storybook’s main concepts and discover additional resources to help you grow and maintain your Storybook.`